backend/remote-state/consul/backend_test.go

```diff
@@ -42,6 +42,7 @@ func TestBackend(t *testing.T) {
 	b := backend.TestBackendConfig(t, New(), map[string]interface{}{
 		"address": srv.HTTPAddr,
 		"path":    fmt.Sprintf("tf-unit/%s", time.Now().String()),
+		"gzip":    true,
```

Comment:
Sure. I did it over in the client tests too, but it doesn't hurt to have it here as well.